oc2pf.md

```diff
@@ -509,390 +489,490 @@ Table 2.3-2 defines the Command Arguments that are allowed for a particular Comm
+* upon completion of all the steps necessary to complete the update and the Actuator commences operations functioning with the new file, OpenC2 Consumers SHOULD respond with Response status code 200.
+* but cannot parse or process the 'update file' Command:
+    * MUST NOT respond with the 200 status code.
+    * SHOULD respond with the 400 status code.
+    * MAY respond with the 500 status code.
+* but do not support the 'update file' Command:
+    * MUST NOT respond with the 200 status code.
+    * SHOULD respond with the 501 status code.
+    * SHOULD respond with 'Command not supported' in the status text.
+    * MAY respond with the 500 status code.
+* but cannot access the file specified in the file Target:
+    * MUST respond with the 500 status code.
+    * SHOULD respond with 'Cannot access file' in the status text.
 
-* but cannot parse or process the Command
-    * MUST NOT respond with a OK/200
-    * SHOULD respond with status code 400
-    * MAY respond with the 500 status code
-* but do not support the 'update file' Command
-    * MUST NOT respond with a OK/200
-    * SHOULD respond with status code 501
-    * SHOULD respond with 'Command not supported' in the status text
-    * MAY respond with status code 500
-* but cannot access the file specified in the file Target
-    * MUST respond with status code 500
-    * SHOULD respond with 'cannot access file' in the status text
-* upon successful parsing and initiating the processing of the 'update file' Command, OpenC2 Consumers MAY respond with Response code 102
-* upon completion of all the steps necessary to complete the update and the Actuator commences operations functioning with the new file, OpenC2 Consumers SHOULD respond with Response code 200
-
-Refer to [Annex A](#annex-a-sample-commands) for sample Commands.
-
----
+# 3 Conformance Statements
+_This section is normative_.
 
-# 3 Conformance statements
-_This section is normative_
-This section identifies the requirements for twenty-two conformance profiles as they pertain to two conformance targets. The two conformance targets are OpenC2 Producers and OpenC2 Consumers (as defined in [Section 1.8](#18-purpose-and-scope) of this specification).
+This section identifies the requirements for twenty-two conformance profiles as they pertain to two conformance targets. The two conformance targets are OpenC2 Producers and OpenC2 Consumers.
 
 ## 3.1 Clauses Pertaining to the OpenC2 Producer Conformance Target
-All OpenC2 Producers that are conformant to this specification MUST satisfy Conformance Clause 1 and MAY satisfy one or more of Conformance Clauses 2 through 11.
+All OpenC2 Producers that are conformant to this specification MUST satisfy Conformance Clause 1 and MAY satisfy one or more of Conformance Clauses 2 through 18.
 
 ### 3.1.1 Conformance Clause 1: Baseline OpenC2 Producer
 An OpenC2 Producer satisfies Baseline OpenC2 Producer conformance if:
```

Comment:
The grammar used throughout these conformance clauses is odd:  ... satisfies conformance if: A.B.C.D **MUST** support ..."

Maybe instead use "To satisfy Baseline OpenC2 Producer conformance an OpenC2 Producer:" A.B.C.D   "**MUST** support ..."

I realize that's a bunch of changes for a grammatical fix but it would definitely read better.  Ideally the 3.1.1.2 language would also be restructured to start with MUST.